src/base/init/config.c

```diff
@@ -670,7 +670,8 @@ static void read_cpu_info(void)
         if (cpuflags && strstr(cpuflags, "tsc")) {
           /* bogospeed currently returns 0; should it deny
            * pentium features, fall back into 486 case */
-          if (strstr(cpuflags, "tsc_reliable")) {
+          if ((strstr(cpuflags, "constant_tsc") && strstr(cpuflags, "nonstop_tsc"))
+                || strstr(cpuflags, "tsc_reliable")) {
```

Comment:
Why do you need tsc_reliable?

Reply:
You're right, it's unnecessary. I'll remove it.